### ISSUE TITLE
Thumbnails missing alt text

### ISSUE THREAD

**Issue by dotherightthing** [dotherightthing/wpdtrt-gallery#63]
![Screen Shot 2019-09-09 at 12 58 24 AM](https://user-images.githubusercontent.com/638871/64488519-fe054480-d29c-11e9-8e54-7789382873d9.png)

**Comment by dotherightthing** [comment]
This also impacts the surrounding link as this is the only content within it

**Comment by dotherightthing** [comment]
Caused by #66
